pkg/config/workloads.go

```diff
@@ -46,10 +59,22 @@ type WorkloadConfig struct {
 type Resources struct {
 	// Specifies the number of CPU shares this pod has access to.
 	CPUShares uint64 `json:"cpushares,omitempty"`
+	// Specifies the CPU quota this pod is limited to.
+	CPUQuota int64 `json:"cpuquota,omitempty"`
+	// Specifies the CPU period to use for these workloads
+	CPUPeriod uint64 `json:"cpuperiod,omitempty"`
```

Comment:
I wonder if mentioning units here would be useful for a reference.

Reply:
That's a good point, we should so there's no confusion. I'll add that in there